python/google/protobuf/pyext/message.cc

```diff
@@ -1055,13 +1055,15 @@ int InternalDeleteRepeatedField(
 
   if (PySlice_Check(slice)) {
     from = to = step = slice_length = 0;
-    PySlice_GetIndicesEx(
 #if PY_MAJOR_VERSION < 3
+    PySlice_GetIndicesEx(
```

Comment:
Yes. They should be the same, except the #if/#else/#endif are moved out.


#if xxxx
foo(XXX)
#else
foo(YYY)
#endif

Instead of
foo(
#if xxxx
XXX
#else
YYY
#endif
)